CompositeUi/src/api/graphql/GraphQl.js

```diff
@@ -49,7 +49,11 @@ class GraphQl {
     );
 
     this.buildGraphQlResponse = (response, dispatch) => (
-      response.catch(() => {
+      response.catch((error) => {
+        if (__DEV__) {
```

Comment:
Don't know if this is the best approach. We should only logout when session expires. When server returns unauthorized we should just show the message. We can keep this approach for debugging temporarily, but we have to implement the other solution